test/nettests/utils.hpp

```diff
@@ -0,0 +1,48 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
```

Comment:
The name of this file *may* be a bit misleading; it's not the header for `test/nettests/utils.cpp`